bootloader/src/pyi_utils.c

```diff
@@ -887,6 +915,10 @@ pyi_utils_create_child(const char *thisfile, const int argc, char *const argv[])
     #endif
 
     pid = fork();
+    if (pid < 0) {
+        VS("LOADER: fork(): %s\n", strerror(errno));
```

Comment:
Please be more verbose in this error message. What should the reader learn if he/she sees this error(?) message?